### PR TITLE
Revamp the AOR's page of Platform

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
As a team we defined our purpose, goals/actions and our principles. In parallel in the same page we define how we work and how you can interact with us.

### CODE REVIEW

**Review by fmartingr — DISMISSED** [review]

**Review by mirshahriar — DISMISSED** [review]

File: operations/research-and-development/organization/cloud_platform.md [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+- Ensuring provisioning, rolling out and controlling the workspaces fleet with an automated manner (eg. provisioner, k8s operator)
+- Improving developer productivity with providing self-served capabilities for testing envs in cloud developer platform (eg. [spinwick](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#spinwick), cloud plugin)
+- Ensuring and supporting other teams for cloud readiness (eg. Mattermost migrations, database connection pooling)
+
+### Meetings
+
+| Topics                             | Meeting                    | Participants                    | Cadence |
+|------------------------------------|----------------------------|---------------------------------|---------|
+| Triage & Planning                  | Platform Planning          | SRE, Platform                   | Tuesday |
+| Cloud & Growth Sync                | Cloud Engineering          | SRE, Release, Platform, Growth  | Weekly  |
```
Comment (gabrieljackson) [inline operations/research-and-development/organization/cloud_platform.md]: Did we want to have `Weekly` for the Cadence for both of these?

File: operations/research-and-development/organization/cloud_platform.md [hunk 2]
```diff
@@ -1,11 +1,51 @@
-# Cloud Platform
+# Platform (Cloud)
 
-The Cloud Platform team is responsible for our Mattermost Cloud SaaS technology.
+### Who We Are
+Platform (Cloud) team is responsible for the underline services which support the orchestration of Mattermost Cloud in terms of provisioning, rolling out and controlling customer's fleet workspaces. In parallel the team boosts the developer productivity using the same services to improve the cloud developer platform.
```
Comment (gabrieljackson) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
Platform (Cloud) team is responsible for the underlying services which support the orchestration of Mattermost Cloud in terms of provisioning, rolling out, and updating the customer Mattermost server fleet. In parallel the team boosts developer productivity using the same services to improve the cloud developer experience.
```

**Review by gabrieljackson — DISMISSED** [review]
Just a few comments/suggestions and then LGTM.

File: operations/research-and-development/organization/cloud_platform.md [hunk 3]
```diff
@@ -1,11 +1,51 @@
...
 
-## Areas of Ownership
+### Vision
+Enable everyone to build and deploy with high velocity in a self-served manner.
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
Enable everyone to build and deploy with high velocity in a self-serve manner.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 4]
```diff
@@ -1,11 +1,51 @@
...
+- Influence Cloud readiness
+- Input driven decision making
+- Encourage best practices and keep quality standards high
+- Keep experimenting
+- Keep documentation fresh
+
+### Principles
+- Balance stability and experimentation
+- Embrace teamwork
+- Leave the room better than it was
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md:19]:
```suggestion
- Leave things better than we found them
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 5]
```diff
@@ -1,11 +1,51 @@
...
+- Data/Feedback driven to find the best solution
+- Distributed systems, distributed team
+- Be curious and keep learning
+- Share knowledge and seek feedback
+
+### Ask for Help
+
+If you need our assistance please use the following follow the [General Workflow](#general-workflow)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
If you need our assistance, please follow the [General Workflow](#general-workflow)
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 6]
```diff
@@ -1,11 +1,51 @@
...
+
+We can also be reached out:
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
More ways to reach us:
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 7]
```diff
@@ -1,11 +1,51 @@
...
+1. For support questions related to production you can reach us in   `~cloud-support ` channel.
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
1. For support questions related to production you can reach us in the `~cloud-support ` channel.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 8]
```diff
@@ -1,11 +1,51 @@
...
+2. For all the other questions you can reach us in `~cloud-platform-team` channel
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
2. For all the other questions you can reach us in the `~cloud-platform-team` channel.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 9]
```diff
@@ -1,11 +1,51 @@
...
+
+### How we work
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
### How We Work
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 10]
```diff
@@ -1,11 +1,51 @@
...
+
+#### General Workflow
+1. Mattermost members open new issues in [Platform Board](https://mattermost.atlassian.net/jira/software/c/projects/MM/boards/46)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
1. Mattermost staff open new issues in the [Platform Board](https://mattermost.atlassian.net/jira/software/c/projects/MM/boards/46).
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 11]
```diff
@@ -1,11 +1,51 @@
...
+2. Every new issues goes through an Issue Triage weekly
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
2. Every new issue goes through an Issue Triage weekly.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 12]
```diff
@@ -1,11 +1,51 @@
...
+
+### Areas of Ownership
+The team regularly works on the following tasks, in the order of priority:
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
The team regularly works on the following tasks, listed in order of priority:
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 13]
```diff
@@ -1,11 +1,51 @@
...
+- Ensuring provisioning, rolling out and controlling the workspaces fleet with an automated manner (eg. provisioner, k8s operator)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md:41]:
```suggestion
- Ensuring that the workspace fleet is provisioned, rolled out, managed and automated (e.g. provisioner, k8s operator)
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 14]
```diff
@@ -1,11 +1,51 @@
...
+- Improving developer productivity with providing self-served capabilities for testing envs in cloud developer platform (eg. [spinwick](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#spinwick), cloud plugin)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
- Improving developer productivity by providing self-serve capabilities for testing environments within the cloud developer platform (e.g. [spinwick](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#spinwick) cloud plugin.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 15]
```diff
@@ -1,11 +1,51 @@
...
+- Ensuring and supporting other teams for cloud readiness (eg. Mattermost migrations, database connection pooling)
```
Comment (cwarnermm) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
- Ensuring and supporting other teams for cloud readiness (e.g. Mattermost migrations, database connection pooling)
```

**Review by cwarnermm — COMMENTED** [review]
Great update, @spirosoik! I've provided editorial feedback for you to consider.

File: operations/research-and-development/organization/cloud_platform.md (hunk 1 above)
Comment (spirosoik) [reply operations/research-and-development/organization/cloud_platform.md]: Should we add the one you have as a team on Thursdays? what's the cadence of this and how this meeting is called?

**Comment by spirosoik** [comment]
@fmartingr @mirshahriar could you please review again?

File: operations/research-and-development/organization/cloud_platform.md [hunk 16]
```diff
@@ -1,11 +1,51 @@
-# Cloud Platform
+# Platform (Cloud)
 
-The Cloud Platform team is responsible for our Mattermost Cloud SaaS technology.
+### Who We Are
+Platform (Cloud) team is responsible for the underlying services which support the orchestration of Mattermost Cloud in terms of provisioning, rolling out, and updating the customer Mattermost server fleet. In parallel the team boosts developer productivity using the same services to improve the cloud developer experience.
```
Comment (mirshahriar) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
Platform (Cloud) team is responsible for the underlying services which support the orchestration of Mattermost Cloud in terms of provisioning, rolling out, and updating the customer's Mattermost server fleet. In parallel, the team boosts developer productivity using the same services to improve the cloud developer experience.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 17]
```diff
@@ -1,11 +1,51 @@
...
+- Distributed systems, distributed team
+- Be curious and keep learning
+- Share knowledge and seek feedback
+
+### Ask for Help
+
+If you need our assistance, please follow the [General Workflow](#general-workflow)
+
+More ways to reach us:
+1. For support questions related to production you can reach us in the `~cloud-support ` channel.
```
Comment (mirshahriar) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
1. For support questions related to production, you can reach us in the `~cloud-support ` channel.
```

File: operations/research-and-development/organization/cloud_platform.md [hunk 18]
```diff
@@ -1,11 +1,51 @@
...
+2. For all the other questions you can reach us in the `~cloud-platform-team` channel.
```
Comment (mirshahriar) [inline operations/research-and-development/organization/cloud_platform.md]:
```suggestion
2. For all the other questions, you can reach us in the `~cloud-platform-team` channel.
```

**Review by mirshahriar — DISMISSED** [review]
LGTM

**Comment by spirosoik** [comment]
@mirshahriar @fmartingr just added the recommendations. let's finalise it and merge it.

**Review by mirshahriar — APPROVED** [review]

**Review by fmartingr — APPROVED** [review]

**Pull request merged**